src/Schema/TableDiff.php

```diff
@@ -250,10 +297,30 @@ public function getDroppedColumns(): array
         return array_values($this->removedColumns);
     }
 
-    /** @return array<string,Column> */
+    /**
+     * @internal Use {@see getModifiedColumns()} instead.
```

Comment:
```suggestion
     * @deprecated Use {@see getModifiedColumns()} instead.
```

Reply:
I put it internal because it's widely used in the tests and causes psalm errors if marked deprecated

```
ERROR: DeprecatedMethod - tests/Schema/AbstractComparatorTestCase.php:745:42 - The method Doctrine\DBAL\Schema\TableDiff::getRenamedColumns has been marked as deprecated (see https://psalm.dev/001)
        self::assertCount(0, $tableDiff->getRenamedColumns());
```

Reply:
But that does not make sense. If you want that calling code should switch to a different method, that's by definition a deprecation.

And we are allowed to call our own deprecated methods. In fact, we need to do that sometimes in order to not lose coverage or break BC.

Have a look at psalm.xml. We have a huge list of deprecated symbols that we allow explicitly. You're free to exended it.